app/web/routes.ts

```diff
@@ -190,6 +191,8 @@ export const howToMakeGreatProfileUrl =
   "https://help.couchers.org/hc/couchersorg-help-center/articles/1725919197-how-do-i-create-a-great-profile";
 export const helpCenterReportContentURL =
   "https://help.couchers.org/hc/couchersorg-help-center/articles/1715125657-how-do-i-report-someone-or-something-that-violates-the-community-guidelines-or-terms-of-use-to_u";
+export const helpCenterPrivateFeedbackUrl =
+  "https://help.couchers.org/hc/couchersorg-help-center/articles/1747589427-providing-private-feedback-when-leaving-a-reference";
```

Comment:
We should also add a link to the page that explains how to write a good reference (I'm not sure if I'm doing this right!):

```suggestion
  "https://help.couchers.org/hc/couchersorg-help-center/articles/1747589427-providing-private-feedback-when-leaving-a-reference";
export const howToLeaveGoodReferenceUrl = 
  "https://help.couchers.org/hc/couchersorg-help-center/articles/1717360836-how-to";
```